### PR TITLE
Fixes #23233 - updated yum_max_speed example to reflect to_bytes parsing

### PR DESCRIPTION
**Pull request by chris1984**
Updated the init.pp description too since passing Kb fails with an error, as its the to bytes conversion is not happy with uppercase.

### CODE REVIEW

**Comment by ekohl** [comment]
If pulp understands these strings, why do we even convert them to bytes in the first place? I'd be happy with removing `to_bytes` and enforcing a regex that Pulp understands so we never accept invalid input to begin with.

**Comment by chris1984** [comment]
@ekohl Pulp only understands bytes it was @sean797 who wanted to add the converter in. The converter is the thing that bombs out with Kb instead of kb... do you want to keep it in still since pulp only understands bytes

File: templates/yum_importer.json.erb [hunk 1]
```diff
@@ -20,7 +20,7 @@
     "proxy_password": null,
     <% end %>
     <% unless [nil, :undefined, :undef, ''].include?(scope['pulp::real_yum_max_speed']) -%>
-    "max_speed": <%= scope['pulp::real_yum_max_speed'] %>,
+    "max_speed": "<%= scope['pulp::real_yum_max_speed'] %>",
```
Comment (ekohl) [inline templates/yum_importer.json.erb]: If pulp only understands bytes, this change should not be needed.

File: manifests/init.pp [hunk 2]
```diff
@@ -186,7 +186,7 @@
 #
 # $proxy_password::             Proxy password for authentication
 #
-# $yum_max_speed::              The maximum download speed for a Pulp task, such as a sync. (e.g. "4 Kb" or 1Gb")
+# $yum_max_speed::              The maximum download speed for a Pulp task, such as a sync. (e.g. "4 kb" or 1gb")
```
Comment (ekohl) [inline manifests/init.pp]: to_bytes is weird. `k[bB]` is used, all the others use upper case prefixes: `M[Bb]` `G[Bb]` etc.

**Comment by chris1984** [comment]
@ekohl updated.

**Comment by chris1984** [comment]
@ekohl updated

**Comment by ekohl** [comment]
Thanks!

**Pull request merged**